pid_error:
  MIN_DISTANCE: 0.1
  MAX_DISTANCE: 30.0
  DESIRED_DISTANCE: .75

  CONTROL_DELAY_ESTIMATE: 0
  minLookDistance: 1
  lookDistanceMultiplier: 1

  currentMode: "center"
  currentEnumMode: 0
  currentGapAngle: 0

control:
  KP: 0.04
  KI: 0.000
  KD: 0.04

  MAX_VEL: 2
  MIN_VEL: 1
  MAX_STEERING_ANGLE: 20
  N: 1
  K: 0.5
